algotaurus.py

```diff
@@ -84,6 +86,7 @@ def __init__(self, labyr):
         """labyr: Labyrinth object
         """
         self.labyr = labyr.labyr
+        self.success = False
```

Comment:
Add a parameter which comes True only if the robot successfully reaches the exit